build/grafana/dashboards/live-network-old.json

```diff
@@ -0,0 +1,2768 @@
+{
+  "annotations": {
```

Comment:
what's the difference between old and new?

Reply:
In addition, please move it to dashboards in root dir

Reply:
And make naming more precise. This is too abstract

Reply:
I will delete this old one, was kept for reference.